flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -60,6 +60,13 @@
 		.withDescription("Service account that is used by jobmanager within kubernetes cluster. " +
 			"The job manager uses this service account when requesting taskmanager pods from the API server.");
 
+	public static final ConfigOption<String> TASK_MANAGER_SERVICE_ACCOUNT =
+		key("kubernetes.taskmanager.service-account")
```

Comment:
I believe we also need to update the `native-kubernetes.md#RBAC` section. At least, we need to tell users they could set the service account for TaskManager via `kubernetes.taskmanager.service-account`. And it is necessary only Kubernetes HA mode,.